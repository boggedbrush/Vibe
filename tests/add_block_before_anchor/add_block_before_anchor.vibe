# VibeSpec: 1.2
patch_type: add_block
file: hello.py
position: before
anchor: "^class Greeter"
--- code: |
  # >>> Inserted before Greeter class
  def helper():
      return True
