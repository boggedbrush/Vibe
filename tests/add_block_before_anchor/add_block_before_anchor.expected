# hello.py
def greet(name):
    print(f"Hello, {name}!")

# >>> Inserted before Greeter class
def helper():
    return True

class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')
